owenhallgren/stretch-project
# Feature/style updates

## What does this PR do?

- Adjust spacing in bottom of Cards
- Reposition homepage header and sort dropdown
- Adjust scrollbar color
- Add hover states to all buttons and icons
- Add title properties to dashboard buttons to make them more intuitive

## How should it be tested in the UI?

- After starting back end API, run ```npm start``` on FE repo
- Perform a full regression test and ensure everything looks good, and all features function as expected
